src/modules/dashboard/components/CoinMachine/RemainingDisplayWidget/RemainingDisplayWidget.tsx

```diff
@@ -66,21 +78,29 @@ const MSG = defineMessages({
     id: 'dashboard.CoinMachine.RemainingDisplayWidget.comeBackTitle',
     defaultMessage: 'Come back in...',
   },
+  soldOut: {
+    id: 'dashboard.CoinMachine.RemainingDisplayWidget.tokensTypeFooterText',
```

Comment:
The id here should have `.soldOut` instead of `.tokensTypeFooterText`, right?

Reply:
good catch!